minitest/redshift_plugin.rb

```diff
@@ -0,0 +1,23 @@
+require "minitest"
+
+module Minitest
+  def self.plugin_redshift_options(opts, options)
+    opts.on "--redshift REDSHIFT_URL", "Configure Redshift connection string" do |redshift_url|
```

Comment:
Can we use `ENV["REDSHIFT_URL"]` instead, which would allow us to remove this file?